website/content/accessibility/deaf-and-hard-of-hearing/conference-best-practices.md

```diff
@@ -0,0 +1,141 @@
+
+### Registration and CFP form
+
+Include a field asking if accommodations are needed (yes/no) and a free-form text box to specify. Everyone's needs differ; it's hard to discern from a fixed form/dropdown.
+
+### Program Information
+
+On the program page, include session accessibility information, such as whether captioning will be provided and in which language. Also, mention that interpreters will be available upon request if requested in advance, and specify which sign language (e.g., American Sign Language, International Sign Language, etc.).  
+
+## Pre-conference best practices 
+
+### Reach out to attendees/speakers before the event
+
+* Set up notifications or monitor registration and CFP forms for accommodation needs and ask for clarification if needed.
+* Connect attendees with sign language interpreters to ensure they understand communication needs and style.
+* For dedicated interpreters (see the [sign language interpreters](#sign-language-interpreters) section), provide both sides with contact information to communicate directly during the event should they run late or if a meeting place changes.  
+
+### Connect us with our peers
+
+Provide a way for deaf and hard of hearing attendees and speakers or people with other disabilities to connect for peer support and potentially pool resources (e.g., an event Slack channel). 
+
+## Day of best practices
+
+Needs vary. Interpreters, captions, or both may be necessary for different individuals.
+
+### Reserved seating at the front
+
+Have some reserved seating at the front for deaf and hard of hearing attendees. Many rely on lip-reading, so you want to be sure they can see the speakers well. And please ask speakers not to wear masks while presenting and provide social distancing opportunities instead. 
+
+### Sign language interpreters
+
+Provide interpreters upon request and in advance. We also recommend having two sign language interpreters on-call to accommodate last-minute registrants or miscommunications. Ask interpreter agencies about their cancellation policies — these can vary significantly. A week's notice, 48, or even 24 hours with a fee, are all possible, depending on the agency. 
+
+Obviously, a big conference like KubeCon has more resources than a local community event — we get that. Since we also want to be included in smaller events, we'll offer various options. To be clear, the more cost-effective options aren't as inclusive, meaning we'll miss out on many opportunities our hearing peers benefit from. Do only go the more cost-effective route if the alternative is no accessibility. We deserve to be included and appreciate your support. 
+
+Sign language interpreters are the most valuable but also the most expensive accommodation. Incorporate the cost into the overall conference cost. It should be no different than including audio/video equipment, swag, food, refreshments, etc. into your budget. 
```

Comment:
I done see a recommendation to put a call out for community members that can serve as interpreters for local events that lack the budget, is this not recommended due to quality concerns?

Reply:
Same here, good point thank you.  I reqested that DHOH group make and issue and follow up with an addtional PR.